### ISSUE TITLE
Wrong T-s output when inputs are two-phase

### ISSUE THREAD

**Issue by babaksamareh** [CoolProp/CoolProp#194]
I think there is something wrong with how the code calculates properties based on T-s. I compared version 5 against 4.2.5, and here is what I found:

Inputs are T = 300 C and s = 4 KJ/kgK

Output of v4.2.5:
Phase: two-phase
P = 85.8791 bar (Correct)
v = 0.00756 m3/kg (correct)
u = 1771.909 KJ/kg (correct)
Q = 30.392% (correct)
However, speed of sound and Cv values for liquid phase are wrong.

Output of v5.0
Phase: not detected
and none of the properties are returned.

Basically, v5.0 cannot solve based on T-s when in the two-phase zone. Any ideas?

PS: v5.0 can successfully solve all the properties and the phase if I use P-s
